### PR TITLE
Feature/include exclude

### PR DESCRIPTION
**Pull request by oonishi870**
add --include/--exclude option
Specify the test case name using a regular expression.

### CODE REVIEW

File: shunit2 [hunk 1]
```diff
@@ -1339,6 +1339,50 @@ _shunit_escapeXmlData() {
   |command sed 's/&/\&amp;/g;s/</\&lt;/g;s/>/\&gt;/g;s/"/\&quot;/g'";s/'/\&apos;/g"
 }
 
+
+arg_parse() {
+    local opt optarg 
+
+    while getopts i:x:-: opt; do
+        # OPTARG を = の位置で分割して opt と optarg に代入
```
Comment (williamdes) [inline shunit2:1347]: can this be translated to english ?

**Comment by williamdes** [comment]
Can you add a test please ?

**Comment by oonishi870** [comment]
Understood. I will make the changes and resubmit the pull request.

**Comment by oonishi870** [comment]
Understood. I will make the changes and resubmit the pull request.

**Comment by williamdes** [comment]
> Understood. I will make the changes and resubmit the pull request.

Next time: no need to re-submit, you can push more changes or write it completely without closing a PR ;)

**Pull request closed without merging**